tchannel-crossdock/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM maven:3.3-jdk-8
+EXPOSE 8080-8082
```

Comment:
kept it for parity with yarpc/jaeger crossdock tests, but only 80/81 are used in practice